### PR TITLE
quiz

### PR DESCRIPTION
**Pull request by sevfitz**

### CODE REVIEW

**Comment by sevfitz** [comment]
Stringified and parsed after the bell rang (it wasn't passing when I submitted...)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+sudo: false
+services:
+- mongodb
+addons:
+  apt:
+    sources:
+    - mongodb-3.4-precise
+    packages:
+    - mongodb-org-server
```
Comment (martypdx) [inline .travis.yml]: Don't need mongo for this quiz

File: lib/vanilla-http.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        res.setHeader('Content-Type', 'application/json');
+        const obj = {
+            'name': 'sevfitz',
+            'since': '2-2-17'
+        };
+        res.end(JSON.stringify(obj));
+
+    }
+    else if (req.method === 'GET') {
+        res.end('Hellow World');
```
Comment (martypdx) [inline lib/vanilla-http.js]: requirement was: `<p>Hello World</p>`

File: tests/vanilla-http-test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+const app = require('../lib/vanilla-http');
+
+describe('/coder', () => {
+    const request = chai.request(app);
+    it('responds with json of the object', done => {
+        request.get('/coder')
+            .end((err, res) => {
+                if (err) done(err);
+                assert.deepEqual(JSON.parse(res.text), {
```
Comment (martypdx) [inline tests/vanilla-http-test.js]: you _could_ use res.body as content type is `application/json`

**Pull request merged**